build.sh

```diff
@@ -60,9 +60,13 @@ function ensureCMakeRan {
     mkdir -p "${LIBRMM_BUILD_DIR}"
     if (( RAN_CMAKE == 0 )); then
         echo "Executing cmake for librmm..."
-        cmake -DCMAKE_INSTALL_PREFIX="${INSTALL_PREFIX}" \
+        cmake -GNinja \
+              -DCMAKE_INSTALL_PREFIX="${INSTALL_PREFIX}" \
               -DCUDA_STATIC_RUNTIME="${CUDA_STATIC_RUNTIME}" \
               -DPER_THREAD_DEFAULT_STREAM="${PER_THREAD_DEFAULT_STREAM}" \
+              -DCMAKE_C_COMPILER_LAUNCHER="${CMAKE_C_COMPILER_LAUNCHER}" \
```

Comment:
I don't think we need to explicitly propagate down the compiler launcher environment variables to CMake. Starting in 3.17 it would do that itself:
https://cmake.org/cmake/help/latest/envvar/CMAKE_LANG_COMPILER_LAUNCHER.html